config/iam_policy_templates/matrix-service-api-lambda.json

```diff
@@ -16,6 +16,17 @@
         "lambda:InvokeFunction"
       ],
       "Resource": "arn:aws:lambda:us-east-1:861229788715:function:dcp-matrix-service-driver-dev"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucket",
```

Comment:
It's a public read bucket. Why is this policy needed?

Reply:
Good call. Removing